test/TestMail.hs

```diff
@@ -3,24 +3,29 @@
 
 module TestMail where
 
+import Types
+import Storage.Notmuch (addTags, removeTags, tagItem)
+
+import Notmuch (mkTag, tagMaxLen)
+
```

Comment:
can you please put "local" deps at bottom of import list?  I like things to run downwards from less specific to more specific imports :)